### PR TITLE
ciba push

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** ciba push

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md](https://github.com/okta/okta-developer-docs/blob/80a2bcb086a78b7eee9f18462f90ec8c1e6db4f2/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fauthenticators-admin%2Findex.md) | [69](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0a5e7c6c-8ec2-46f2-b58f-02bf2c3dbfa8) | 58  | 83  | 77  | 62  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/oidc/index.md](https://github.com/okta/okta-developer-docs/blob/80a2bcb086a78b7eee9f18462f90ec8c1e6db4f2/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Foidc%2Findex.md) | [71](https://okta.acrolinx.cloud/api/v1/checking/scorecards/55c22d2c-3e99-43a5-9e3e-d14ccb85b5a2) | 71  | 66  | 58  | 89  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/push-providers/index.md](https://github.com/okta/okta-developer-docs/blob/80a2bcb086a78b7eee9f18462f90ec8c1e6db4f2/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpush-providers%2Findex.md) | [67](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9060d8fb-66cc-4944-8479-69649236e7bf) | 84  | 51  | 78  | 50  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/80a2bcb086a78b7eee9f18462f90ec8c1e6db4f2/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2022-okta-identity-engine%2Findex.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f00e6e90-9161-4df2-ad52-65fdd48e8257) | 54  | 76  | 43  | 99  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/80a2bcb086a78b7eee9f18462f90ec8c1e6db4f2/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | [71](https://okta.acrolinx.cloud/api/v1/checking/scorecards/772e2f45-af08-4018-a378-0be75330ee35) | 60  | 77  | 46  | 99  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.3708ba8c-6c16-405b-9683-65bb73916b7a?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/reference/api/push-providers/index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The Push Providers API supports the following Authorization Schemes:
 * SSWS - [API tokens](/docs/reference/core-okta-api/#authentication)
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/push-providers/index.md:16]: @vanngo-okta @jane-kuang-okta should we completely remove this file from the docs?

**Review by brentschaus-okta — APPROVED** [review]
LGTM other than my question about removing the push-providers content altogether

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md](https://github.com/okta/okta-developer-docs/blob/8a25112eb410811d84f1660b9840cae4a43d5c94/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fauthenticators-admin%2Findex.md) | [69](https://okta.acrolinx.cloud/api/v1/checking/scorecards/fcd34a3f-3446-4ace-b874-5ce15fdc13e8) | 56  | 82  | 75  | 63  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/oidc/index.md](https://github.com/okta/okta-developer-docs/blob/8a25112eb410811d84f1660b9840cae4a43d5c94/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Foidc%2Findex.md) | [71](https://okta.acrolinx.cloud/api/v1/checking/scorecards/260de7ca-13e9-4f11-96ee-cd425e63af8f) | 71  | 66  | 58  | 89  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/push-providers/index.md](https://github.com/okta/okta-developer-docs/blob/8a25112eb410811d84f1660b9840cae4a43d5c94/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpush-providers%2Findex.md) | [67](https://okta.acrolinx.cloud/api/v1/checking/scorecards/201442fc-bc0b-4627-a76c-fcd5baf0a764) | 84  | 51  | 78  | 50  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/8a25112eb410811d84f1660b9840cae4a43d5c94/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2022-okta-identity-engine%2Findex.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8ce24ec4-35dc-45b6-bd64-6a4834c8535c) | 54  | 76  | 43  | 99  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/8a25112eb410811d84f1660b9840cae4a43d5c94/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | [71](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6d27161b-92fa-47ea-ba45-717817292721) | 60  | 77  | 46  | 99  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.26eb6321-6f76-4753-b633-34019be39b6f?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by vanngo-okta — APPROVED** [review]

**Pull request merged**